space-plugins/nuxt-starter/app.config.ts

```diff
@@ -1,14 +1,8 @@
 export default defineAppConfig({
+	type: 'space-plugin',
```

Comment:
This was outdated. We've moved `type` to the top level of app config in a recent PR.